raiden/network/proxies/token_network.py

```diff
@@ -530,8 +542,8 @@ def close(
             self,
             channel_identifier: typing.ChannelID,
             partner: typing.Address,
-            nonce: typing.Nonce,
             balance_hash: typing.BalanceHash,
+            nonce: typing.Nonce,
```

Comment:
also here why the change of the nonce argument position?

Reply:
This function here had the nonce order changed but you did not add the `channel_identifier` [here](https://github.com/raiden-network/raiden/pull/2018/files#diff-d02574a7a7eca826412780e80a6326a3R581) into the actual proxy call.